homeassistant/components/growatt_server/__init__.py

```diff
@@ -64,6 +64,9 @@ def get_device_list_classic(
 
     user_id = login_response["user"]["id"]
 
+    # Legacy support: DEFAULT_PLANT_ID ("0") triggers auto-selection of first plant.
+    # Modern config flow always sets a specific plant_id, but old config entries
+    # from earlier versions may still have plant_id="0".
```

Comment:
It would be cool if we could migrate this at some point, as we can fetch all the plants in the migration logic and just make sure everyone is on the same track

Reply:
Yep, you understood what I meant. I think the benefit is future stability. Now, a contributor who wants to contribute to the integration needs to know that there are plant IDs with "0". If they ignore that, let's say there's a new endpoint that takes the plant ID and the user sources it from the entry data. That'd work on their machine. In a world where you and me left the project or we are idk, on a world trip, someone else wouldn't know that the plant ID can be "0", which would break this feature. So by making this constant we at least make it consistent and more predictable :)